contrib/util/ccda_import/import_ccda.php

```diff
@@ -88,7 +90,8 @@ function outputMessage($message): void
 {
     echo("\n");
     echo $message;
-    file_put_contents("log.txt", $message, FILE_APPEND);
+    $file = $GLOBALS['OE_SITE_DIR'] . "/documents/temp/log.txt";
+    file_put_contents($file, $message, FILE_APPEND);
```

Comment:
here's a good place where can use the above flag to save the log in the correct place (ie. in web page mode to /documents/temp/log.txt and in original command line mode in the directory that the script is run in)